### ISSUE TITLE
"Как может быть ещё лучше? ". → «Как может быть ещё лучше? «. 

### ISSUE THREAD

**Issue by dy** [typograf/typograf#379]
Может имело бы смысл попарной детекции кавычек?

**Comment by hcodes** [comment]
Имеет смысл, нужно поправить, спасибо за баг.

**Comment by denisnarush** [comment]
А разве после знака вопроса должен быть пробел перед кавычкой?

**Comment by dy** [comment]
Нет, но это вид опечатки. Типограф исходит из того что опечаток нет?

**Comment by denisnarush** [comment]
@dy Ну в этом случае если типограф уберет опечатку то проблема решится. Т.е. я бы исходил изначально из того что создало проблему, а затем уже правила бы применял.
